infra/Makefile

```diff
@@ -0,0 +1,10 @@
+default: plan
+
+init:
+	cd envs/$(DEPLOYMENT_STAGE) && make init
```

Comment:
Nit: I like this syntax more.
```suggestion
	make -C envs/$(DEPLOYMENT_STAGE) init
```

Reply:
Leaving as is. Feel free to change on another PR if you have strong feelings